src/i18n/rpi-imager_zh.ts

```diff
@@ -595,37 +515,37 @@
     <message>
         <location filename="../OptionsServicesTab.qml" line="36"/>
         <source>Enable SSH</source>
-        <translation type="unfinished">开启 SSH</translation>
+        <translation>开启 SSH</translation>
     </message>
     <message>
         <location filename="../OptionsServicesTab.qml" line="52"/>
         <source>Use password authentication</source>
-        <translation type="unfinished">使用密码登录</translation>
+        <translation>使用密码登录</translation>
     </message>
     <message>
         <location filename="../OptionsServicesTab.qml" line="63"/>
         <source>Allow public-key authentication only</source>
-        <translation type="unfinished">仅使用公钥登录</translation>
+        <translation>仅使用公钥登录</translation>
     </message>
     <message>
         <location filename="../OptionsServicesTab.qml" line="74"/>
         <source>Set authorized_keys for &apos;%1&apos;:</source>
-        <translation type="unfinished">为用户 “%1” 设置 authorized_keys：</translation>
+        <translation>为用户 “%1” 设置 authorized_keys：</translation>
     </message>
     <message>
         <location filename="../OptionsServicesTab.qml" line="121"/>
         <source>Delete Key</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
I guess this translation still ought to be tagged as `type="unfinished"`? (as well as the one on line 548)